agent.go

```diff
@@ -1060,6 +1062,12 @@ func addMounts(config *configs.Config, fsmaps []hyper.Fsmap) error {
 func newContainerCb(pod *pod, data []byte) error {
 	var payload hyper.NewContainer
 
+	// re-scan PCI bus
+	// looking for hidden devices
+	if err := ioutil.WriteFile(pciBusRescanFile, []byte("1"), pciBusMode); err != nil {
+		agentLog.WithError(err).Warnf("Could not open pci bus rescan file %s: %s", pciBusRescanFile, err)
```

Comment:
`"Could not rescan the PCI bus"` would be more generic and more appropriate. The underlying reason why this is happening will come from `err`.